playbooks/templates/plans/README-plans.md

```diff
@@ -0,0 +1,34 @@
+# Introduction CI Testing Plans
+
+Linux System Roles CI runs [tmt](https://tmt.readthedocs.io/en/stable/index.html) test plans in [Testing farm](https://docs.testing-farm.io/Testing%20Farm/0.1/index.html) with the https://github.com/linux-system-roles/%{{ inventory_hostname }}/blob/main/.github/workflows/tmt.yml GitHub workflow.
+
+The plans/general.fmf plan is a test plan that is general for all roles. It does the following steps:
+1. Provisiones two machines, one used as an Ansible control node, and second used as a managed node.
+2. Does the required preparation on machines.
+3. For the given role and the given PR, runs the general test from https://github.com/linux-system-roles/tft-tests/blob/main/tests/general/test.sh.
+
+The [tmt.yml](https://github.com/linux-system-roles/%{{ inventory_hostname }}/blob/main/.github/workflows/tmt.yml) workflow runs the above plan and uploads the results to our Fedora storage for public access.
+This workflow uses Testing Farm's Github Action [Schedule tests on Testing Farm](https://github.com/marketplace/actions/schedule-tests-on-testing-farm).
+
+## Running Tests
+
+You can run tests locally with the `tmt try` cli.
+
+### Prerequisites
+
+* Install `tmt` as described in [Installation](https://tmt.readthedocs.io/en/stable/stories/install.html).
```

Comment:
Does this require some sort of credentials?

Reply:
No, this is run locally.
Credentials, a Testing Farm API token, would be required for running `testing-farm request`

Reply:
ah - so it will provision two local qemu VMs?  Where does it get the qcow2 images from?

Reply:
Right, two local VMs with the same platform, the platform is defined by matrix in https://github.com/linux-system-roles/.github/pull/64/files#diff-8ea98de8fe4a0e02f755418a7e04df47298bb59ee9096c119ba68905ec8696d3R95

About where images are pulled - no idea. You can provide a custom image URL if you want.